common/src/View/QtUtils.cpp

```diff
@@ -72,6 +73,32 @@ namespace TrenchBroom {
             m_widget->setUpdatesEnabled(true);
         }
 
+        SyncHeightEventFilter::SyncHeightEventFilter(QWidget* master, QWidget* slave) :
+        m_master(master),
+        m_slave(slave) {
+            ensure(m_master != nullptr, "master is not null");
+            ensure(m_slave != nullptr, "slave is not null");
+            
+            m_master->installEventFilter(this);
+        }
+        
+        SyncHeightEventFilter::~SyncHeightEventFilter() {
+            m_master->removeEventFilter(this);
```

Comment:
m_master and m_slave should probably be `QPointer<QWidget>` (this is a weak pointer to QObject that's automatically cleared if the QObject is deleted first).

Reply:
It should just do nothing if either pointer is expired, since this should only happen during the window tear down.

Reply:
Changed in c1d7bbded. Good catch, I'll probably start using `QPointer` more.